stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt

```diff
@@ -1,6 +1,32 @@
 package com.getstream.sdk.chat.utils.extensions
 
 import io.getstream.chat.android.client.models.Channel
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+import io.getstream.chat.android.livedata.ChatDomain
 
 internal val Channel.isDraft: Boolean
     get() = getExtraValue("draft", false)
+
+@InternalStreamChatApi
```

Comment:
A nice method to be exposed to our customers

Reply:
I'll leave it like this for now, but we can expose it later on. (It would also be a good idea to move it to the new package name before doing so, don't want any more API that uses `com.getstream`)